### ISSUE TITLE
Missing tags for newer doxygen versions

### ISSUE THREAD

**Issue by rasky** [mattnotmitt/doxygen-action#39]
Doxygen is now on v1.12.0. This repo is missing tags for versions 1.10, 1.11 and 1.12. Unfortunately, being tags, it cannot be PR'd. Can you please add them? Thanks

**Comment by mattnotmitt** [comment]
Sorry about the wait! I've just released `v1.11.0` and `v1.12.0`. `1.10` is missing as there was no alpine release which has this version still. I can't promise I'll always have the tags in place promptly, but if you want the latest and greatest, you can use `edge`, which will just use the latest alpine image with the latest version of doxygen they have packaged (`v1.13` as of this comment). I'll try to keep on top of it more this year.

**Comment by rasky** [comment]
Yes the problem with using `edge` is that it will unexpectedly change version when a new one is released, and doxygen has a habit of breaking things between releases (eg: changing config file keys, adding new validations that would catch wrong comments that were accepted before, etc.). This in turns means that my CI, that runs doxygen, will unexpectedly start failing one random day. 

I understand there's no simple solution here.
